features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2403,7 +2404,7 @@ CREATE TABLE IDN_WEBHOOK (
     PRIMARY KEY (ID),
     UNIQUE(UUID),
     UNIQUE(TENANT_ID, ENDPOINT)
-)
+) IN TS32K
```

Comment:
Is this needed ?

Reply:
Needed since the endpoint and tenant index exceeds the default storage size.